hooks/useGetBatchClaims.ts

```diff
@@ -38,15 +38,17 @@ export const useGetBatchClaims = () => {
 
   useEffect(() => {
     void fetchClaims();
+
+    const claimsIntervalId = setInterval(() => {
+      void fetchClaims();
+    }, 20000);
+
+    return () => clearInterval(claimsIntervalId);
   }, [fetchClaims]);
 
   const refetch = async () => {
     await fetchClaims();
```

Comment:
We can probably remove this refetch function since we're using the setInterval or we can leave it in since we're changing it anyway

same with the isSuccess ? refetch functionality in claimRewards.tsx. Otherwise I'll remove them when I make changes